mobile/components/SwipeableExpenseRow.js

```
@@ -0,0 +1,69 @@
+import React, { useRef } from 'react';
+import { StyleSheet, View } from 'react-native';
+import ReanimatedSwipeable from 'react-native-gesture-handler/ReanimatedSwipeable';
+import Reanimated, { useAnimatedStyle, interpolate, Extrapolation } from 'react-native-reanimated';
+import { IconButton } from 'react-native-paper';
+
+const SwipeableExpenseRow = ({ children, onSwipeableOpen }) => {
+  const swipeableRef = useRef(null);
+
+  const renderRightActions = (progress, drag) => {
+    const style = useAnimatedStyle(() => {
```

Biome (2.3.14)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)